docs/installation/linux.md

```diff
@@ -672,6 +672,35 @@ chmod a+x powershell-6.0.0-rc-x86_64.AppImage
 dpkg -r powershell_6.0.0-rc-1.ubuntu.16.04_amd64.deb
 ```
 
+## Raspbian
+
+Currently, PowerShell is only supported on Rasbian Stretch.
+
+### Installation
+
+```sh
+# Install prerequisites
+sudo apt-get install libunwind8 libicu57
```

Comment:
In my test, to run PS-RC on Rasbian Stretch, I didn't have to deploy libicu57; only libunwind8.

Reply:
Ok, I'll remove that.